stores/governance/src/main/java/com/bloxbean/cardano/yaci/store/governance/service/LocalGovStateService.java

```diff
@@ -203,6 +186,11 @@ private void handleGovStateQueryResult(GovStateQueryResult govStateQueryResult)
                         .gov_action_index((int) entity.getGovActionIndex())
                         .build()).toList();
 
+        if (latestCursor.isEmpty() || tip.getBlock() > latestCursor.get().getBlock() + MAX_BLOCK_DIFFERENCE) {
```

Comment:
Is this `if` condition required?

Reply:
Because we need the content of the hard fork init and treasury withdrawal, we need to reference the gov_action_proposal table. 
However, we might not need this condition, since we only insert into the two tables local_hard_fork_initiation and local_treasury_withdrawal  when the actions are enacted, and the actions, from the time they are created to the time they are enacted, must take at least more than one epoch. Therefore, the condition syncMode = true might already be sufficient.

Reply:
If it's not required, let's remove it to keep the logic simple.